go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -35,11 +36,13 @@ var (
 	flagCertificateUrl = flag.String("certUrl", "https://example.com/cert.msg", "The URL where the certificate chain is hosted at.")
 	flagValidityUrl    = flag.String("validityUrl", "https://example.com/resource.validity.msg", "The URL where resource validity info is hosted at.")
```

Comment:
I realized that this should be `flagCertificateUrl` and `flagValidityURL` for Go convention. Let's address on this later.